Sources/ArcGISToolkit/Components/Offline/OnDemand/OnDemandMapAreaSelectorView.swift

```diff
@@ -228,8 +223,11 @@ struct OnDemandMapAreaSelectorView: View {
     
     /// The handle view for the map area selector.
     struct Handle: View {
+        let location: HandleLocation
```

Comment:
I wonder if we should call this `HandleOrientation` because "location" and "position" can be a bit overloaded.

```swift
let orientation: HandleOrientation
```